### PR TITLE
Add Control Center Manual and Getting Started Manuals

### PR DESCRIPTION
**Pull request by vertiq-jordan**
Adds the Getting Started with Control Center/Firmware Manuals to help people get spinning with their modules as fast as possible using the Control Center. Added documents are the control center getting started, pulsing module getting started, speed module getting started, and servo module getting started.

The built docs can be found here: https://iqmotion.readthedocs.io/en/feature-control_center_and_getting_started/

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -25,6 +25,14 @@
    modules/vertiq_81xx_family
    modules/fortiq_42XX
 
+.. toctree::
+   :caption: IQ Control Center
+   :hidden:
+
```
Comment (vertiq-jordan) [inline docs/index.rst:31]: add the manuals to the side bar

File: docs/manual/manual_advanced_arming.rst [hunk 2]
```diff
@@ -497,7 +497,7 @@ The examples below cover the arming specific setup steps on a Vertiq module and
 Using throttle regions for arming and disarming is recommended when integrating with flight controllers, as it allows for the simplest integration. 
 
 These examples cover only the additional setup needed to support arming and assume the mode, direction and unit maximums are already set appropriately on the module. 
-For examples of the initial setup of a Vertiq module see the :ref:`control_center_tutorial` tutorial and the :ref:`hobby_fc_tutorial` tutorial. 
+For examples of the initial setup of a Vertiq module see the :ref:`speed_module_start_guide` tutorial and the :ref:`hobby_fc_tutorial` tutorial.
```
Comment (vertiq-jordan) [inline docs/manual/manual_advanced_arming.rst:500]: update ref to somewhere that makes more sense now

File: docs/manual/manual_stow.rst [hunk 3]
```diff
@@ -171,7 +171,7 @@ The second method makes it possible to simply move the module to the position yo
 
 #. Move your module into the desired zero angle. Having a propeller attached to the module may make it easier to determine if you are in the desired orientation.
 #. Connect to the module with the IQ Control Center, and open the General tab.
-#. Click the :ref:`set arrow <control_center_tutorial_spinning_motor>` on the :ref:`Sample Zero Angle <stow_angle_parameters>` entry in the IQ Control Center. This will automatically set the current angle as the zero angle. 
+#. Click the :ref:`set arrow <save_button>` on the :ref:`Sample Zero Angle <stow_angle_parameters>` entry in the IQ Control Center. This will automatically set the current angle as the zero angle.
```
Comment (vertiq-jordan) [inline docs/manual/manual_stow.rst:174]: update ref to somewhere that makes more sense now

File: docs/manual/manual_throttle.rst [hunk 4]
```diff
@@ -109,7 +109,7 @@ Mode, Maximums, and Direction
 To fully understand how a Vertiq module will interpret a throttle command, several key configurations must be covered. 
 The definition of a throttle command is covered in the :ref:`throttle_def` section.
 
-For detailed instructions on setting these parameters, see the :ref:`control_center_tutorial` and :ref:`hobby_fc_tutorial` tutorials.
+For detailed instructions on setting these parameters, see the :ref:`speed_module_start_guide` and :ref:`hobby_fc_tutorial` tutorials.
```
Comment (vertiq-jordan) [inline docs/manual/manual_throttle.rst:112]: update ref to somewhere that makes more sense now

File: docs/modules/get_started_text.rst [hunk 5]
```diff
@@ -1,17 +1,16 @@
 Changing or Updating Firmware
 ####################################
 
-Please follow the instructions found in :ref:`updating_firmware`
+Please follow the instructions found in :ref:`updating_firmware_control_center`
```
Comment (vertiq-jordan) [inline docs/modules/get_started_text.rst:4]: update ref to somewhere that makes more sense now

**Review by iq-fred — APPROVED** [review]

**Pull request merged**